pathfinder/paizo/campaign_setting/inner_sea_magic/ism_classes.lst

```diff
@@ -12,6 +12,9 @@ CLASS:Eidolon (Fey)	PRERACETYPE:Eidolon
 # Class Name		Skill Pts/Lvl	Add INT to Skill?	Class Skill
 CLASS:Eidolon (Fey)	STARTSKILLPTS:6	MODTOSKILLS:YES	CSKILL:Acrobatics|Bluff|Climb|TYPE=Craft|Diplomacy|Disguise|Escape Artist|Fly|Knowledge (Geography)|Knowledge (Nature)|Perception|TYPE=Perform|Sense Motive|Sleight of Hand|Stealth|Swim|Use Magic Device
 
+###Block: Witch ~ Winter Witch
+CLASS:Witch.MOD	PROHIBITSPELL:DESCRIPTOR.Fire	PREABILITY:1,CATEGORY=Special Ability,Winter Witch ~ Ice Magic
+
```

Comment:
PREABILITY should only apply to the PROHIBITSPELL of course. Will fix & resubmit.